beacon-chain/forkchoice/doubly-linked-tree/store.go

```diff
@@ -290,3 +290,18 @@ func (f *ForkChoice) ReceivedBlocksLastEpoch() (uint64, error) {
 	}
 	return count, nil
 }
+
+// SetPayloadRevealBoostRoot sets the root of the block that receives the reveal boost.
+func (s *Store) SetPayloadRevealBoostRoot(root [fieldparams.RootLength]byte) {
```

Comment:
Why do these have to be exported?

Reply:
I intended to use it, but seems we don't need these functions